django/src/rdwatch/utils/worldview_processed/raster_tile.py

```diff
@@ -98,5 +98,7 @@ def get_worldview_processed_visual_bbox(
             if capture.bits_per_pixel != 8:
                 max_bits = 2**capture.bits_per_pixel - 1
                 rgb.rescale(in_range=((0, max_bits),))
+        elif len(scale) == 2:  # scale is an integeter range
```

Comment:
I think you need to update the type hint for this in the function args. Also, same suggestion as above -
```suggestion
        elif isinstance(scale, list) and len(scale) == 2:  # scale is an integeter range
```